internal/notify/notifications.lua

```diff
@@ -524,6 +526,25 @@ NOTIFICATIONS_BY_IDX = {
         adv_fn=curry(get_bar, get_blood, get_max_blood, "Blood", COLOR_RED),
         on_click=nil,
     },
+    {
+        name='autosave',
```

Comment:
this will show up in the `gui/notify` list, so the name is important. maybe `save-reminder` would be a clearer name?